redis/src/bignum.rs

```diff
@@ -0,0 +1,74 @@
+use std::str::from_utf8;
+
+use super::{ErrorKind, RedisError, RedisResult};
+use crate::types::{FromRedisValue, RedisWrite, ToRedisArgs, Value};
+
+macro_rules! invalid_type_error {
```

Comment:
these are copies from types.rs. why not just add the logic there?

Reply:
Moved to types.rs